src/test/kotlin/fi/metatavu/timebank/api/test/functional/tests/VacationRequestsTest.kt

```diff
@@ -0,0 +1,97 @@
+package fi.metatavu.timebank.api.test.functional.tests
+
+import fi.metatavu.timebank.api.test.functional.data.TestDateUtils.Companion.getODT
+import fi.metatavu.timebank.api.test.functional.data.TestDateUtils.Companion.getThirtyDaysAgoThirdWeek
+import fi.metatavu.timebank.api.test.functional.resources.LocalTestProfile
+import fi.metatavu.timebank.api.test.functional.resources.TestMySQLResource
+import fi.metatavu.timebank.api.test.functional.resources.TestWiremockResource
+import fi.metatavu.timebank.test.client.models.RequestStatus
+import fi.metatavu.timebank.test.client.models.VacationRequest
+import fi.metatavu.timebank.test.client.models.VacationType
+import io.quarkus.test.common.QuarkusTestResource
+import io.quarkus.test.junit.QuarkusTest
+import io.quarkus.test.junit.TestProfile
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import java.util.*
+import org.junit.jupiter.api.Assertions.*
+import java.time.LocalDate
+
+/**
+ * Tests for Vacations API
+ */
+@QuarkusTest
+@QuarkusTestResource.List(
+    QuarkusTestResource(TestMySQLResource::class),
+    QuarkusTestResource(TestWiremockResource::class)
+)
+@TestProfile(LocalTestProfile::class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class VacationRequestsTest: AbstractTest() {
+
+    /**
+     * Resets Wiremock scenario states before each test
+     */
+    @BeforeEach
+    fun resetScenariosBeforeEach() {
+        resetScenarios()
+    }
+
+    /**
+     * Tests /v1/vacationRequest -endpoint
+     */
+    @Test
+    fun testVacationRequest() {
+        createTestBuilder().use { testBuilder ->
+            testBuilder.manager.vacations.createVacationRequests(
+                vacationRequest = VacationRequest(
+                    id = UUID.fromString("5c2b0646-8e87-4b4e-9b7a-624ca1bf832d"),
```

Comment:
This same UUID is being used in 5 places, assign it to a variable instead of typing by hand so many times. Just to avoid typos and frustration 😆